### PR TITLE
update use case for blog

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

**Comment by Bonczidai** [comment]
tests for this use case need to be moved to correct folder (\content\io\cloudslang\docker\monitoring)

File: content/io/cloudslang/docker/monitoring/mysql/check_mysql_is_up.sl [hunk 1]
```diff
@@ -11,19 +11,18 @@
 # Inputs:
 #   - container - name or ID of the Docker container that runs MySQL
 #   - host - Docker machine host
-#   - port - optional - SSH port - Default: 22
+#   - port - optional - SSH port
 #   - username - Docker machine username
-#   - password - Docker machine password
-#   - privateKeyFile - optional - absolute path to private key file - Default: none
-#   - arguments - optional - arguments to pass to the command - Default: none
```
Comment (oritstone) [inline content/io/cloudslang/docker/monitoring/mysql/check_mysql_is_up.sl]: Why did you remove arguments from the doc. It is one of the inputs...

File: content/io/cloudslang/docker/monitoring/mysql/check_mysql_is_up.sl [hunk 2]
```diff
@@ -49,21 +48,21 @@ flow:
...
-    - characterSet:
+    - character_set:
         required: false
     - pty:
         required: false
     - timeout:
         required: false
-    - closeSession:
+    - close_session:
         required: false
```
Comment (oritstone) [inline content/io/cloudslang/docker/monitoring/mysql/check_mysql_is_up.sl]: missing - agent_forwarding param

File: content/io/cloudslang/docker/monitoring/mysql/check_mysql_is_up.sl [hunk 3]
```diff
@@ -77,22 +76,22 @@ flow:
...
                 required: false
             - pty:
                 required: false
             - timeout:
                 required: false
             - closeSession:
-                required: false
-            - agentForwarding:
+                default: close_session
                 required: false
```
Comment (oritstone) [inline content/io/cloudslang/docker/monitoring/mysql/check_mysql_is_up.sl]: Why do you use the close_session as agent_forwarding?

File: content/io/cloudslang/docker/monitoring/mysql/report_mysql_status.sl [hunk 4]
```diff
@@ -33,11 +33,11 @@ flow:
 
   inputs:
     - container
-    - docker_host
-    - docker_username
-    - docker_password
+    - host
+    - username
+    - password
```
Comment (oritstone) [inline content/io/cloudslang/docker/monitoring/mysql/report_mysql_status.sl]: should be required: false

**Comment by oritstone** [comment]
Opened a PR for moving the test: #211

**Comment by Bonczidai** [comment]
We can merge this PR, right?

**Comment by oritstone** [comment]
@Bonczidai is there still effort in updating the blog?

**Comment by Bonczidai** [comment]
two small fixes are needed:
- the CLI cmd needs to be changed to this:
```
cslang> run --f c:/…/report_mysql_status.sl --i host={host},username={ username},password={password},container={container_id_or_name},mysql_username={mysql_username},mysql_password={mysql_password},email_host={email_host},email_port={email_port},email_sender={email_sender},email_recipient={email_recipient} --cp C:/.../content/
```
- the screenshots of flows needs to be updated

@SamMarkowitz could you please update the blog?

**Comment by SamMarkowitz** [comment]
@Bonczidai do the screenshots need to be reupdated. I just changed them last week. Have there been any changes since then?

**Comment by Bonczidai** [comment]
there are some changes affecting the inputs (input names, required property) on this PR (which is merged - so now they are on master) - I say we should update those screenshots :wink:

**Comment by SamMarkowitz** [comment]
Okay. Will do.

**Comment by Bonczidai** [comment]
thx
